doc/lsst.obs.decam/index.rst

```diff
@@ -0,0 +1,22 @@
+.. _lsst.obs.decam:
+
+###############
+lsst.obs.decam
+###############
+
+.. Paragraph that describes what this Python module does and links to related modules and frameworks.
+
+.. Add subsections with toctree to individual topic pages.
+
```

Comment:
Redo this file to match this example. In particular, note the `Task` section.

https://github.com/lsst/templates/blob/master/project_templates/stack_package/example/doc/lsst.example/index.rst